utils/i2c-tools/Makefile

```diff
@@ -15,17 +15,18 @@ PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=@KERNEL/software/utils/i2c-tools
 PKG_HASH:=57b219efd183795bd545dd5a60d9eabbe9dcb6f8fb92bc7ba2122b87f98527d5
 
-PKG_BUILD_PARALLEL:=1
```

Comment:
let's leave `PKG_BUILD_PARALLEL:=1`

Reply:
I am not sure about it. I tested it and it seems that build in paralel works. I tested it with sixteen jobs in paralel and our server already build this multiple times with eight parallel jobs. I am not sure if this is really needed but I reverted it either way.

Reply:
is this with the vanilla OpenWrt or some custom/internal fork of OpenWrt ?
looking in the code, it doesn't look like this is set by default; see here https://github.com/openwrt/openwrt/blob/master/include/package.mk#L16

to my understanding, this setting typically refers to parallelizing the build of a single package;
it may be that OpenWrt can build several packages in parallel [each being a single build job], but any single package can have multiple job when compiling; 

so, i would say to leave this here, unless this breaks something [else]

Reply:
ah, this was moved lower;
fine by me :)

Reply:
It was with both.. Vanilla openwrt on my machine and Turris on our servers.

I don't like such approach because that can be introduced to every package this way and that just prolongs compilation time. I added it back just to satisfy your request but I don't see a point it having it there. It is possible that some old version of i2c-tools was failing to compile and since then because of this approach this option is dragged with package. It does make sense to me to let it there. If it is standing issue that manifests itself only in some corner cases then it should have been documented near that option. I think that we should drop that, but that is only my opinion.

Reply:
But I don't see point in prolonging this merge request with this.